4_arm.ipynb

```diff
@@ -321,6 +593,19 @@
     "ここで姿勢はx軸, y軸, z軸の順番でroll, pitch, yawの値で回転させ、その値を四元数に変換しています。"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Given the target coordinates of the hand, calculating joint angles of the arm is called \"solving inverse kinematics\".\n",
+    "\n",
+    "First, set the target coordinates. The target of the end effector is position: (x, y, z)=(0.2,0.4,1.0) and orientation: (roll, pitch, yaw)=(180, 90, 0) on the base coordinate \"map\".\n",
```

Comment:
I decided to use embedded quotes to help separate the position/orientation specifications visually within the sentence.

    "First, set the target coordinates. The target of the end effector is \"position: (x, y, z)=(0.2,0.4,1.0)\" and \"orientation: (roll, pitch, yaw)=(180, 90, 0)\" on the base coordinate \"map\".\n",